common4j/src/main/com/microsoft/identity/common/java/platform/AbstractDevicePopManager.java

```diff
@@ -122,6 +122,12 @@ public abstract class AbstractDevicePopManager implements IDevicePopManager {
      */
     private static final int RSA_KEY_SIZE = 2048;
 
+    /**
+     * Seeing this on android 13, we think it's being caused by the new Samsung patch release Build Number : G991BXXS9EWIA
+     * <a href="https://doc.samsungmobile.com/sm-g991b/xeo/doc.html">...</a>
+     */
+    public static final String NEGATIVE_THIRTY_THREE_INTERNAL_ERROR = "internal Keystore code: -33";
```

Comment:
Suggest renaming `NEGATIVE_THIRTY_THREE_INTERNAL_ERROR` to `NEGATIVE_THIRTY_THREE_INTERNAL_ERROR_MSG` (since I think this comes back as the message on the exception object)

Reply:
ack